src/DetailsView/components/details-view-overlay/details-view-overlay.tsx

```diff
@@ -24,7 +24,7 @@ import {
 
 export type DetailsViewOverlayDeps = {
     detailsViewActionMessageCreator: DetailsViewActionMessageCreator;
-    assessmentActionMessageCreator: AssessmentActionMessageCreator;
+    getAssessmentActionMessageCreator: () => AssessmentActionMessageCreator;
```

Comment:
nit: it seems like we're not using this here; we should probably remove it entirely.